template/conftest.py

```diff
@@ -16,19 +16,19 @@
 
 BACKENDS = ["speculos", "ledgercomm", "ledgerwallet"]
 
+DEVICES = ["nanos", "nanox", "nanosp", "all"]
+
 FIRMWARES = [Firmware('nanos', '2.1'),
              Firmware('nanox', '2.0.2'),
              Firmware('nanosp', '1.0.3')]
 
 
 def pytest_addoption(parser):
-    parser.addoption("--backend", action="store", default="speculos")
+    parser.addoption("--device", choices=DEVICES, required=True)
```

Comment:
default="all" would be nice IMO

Reply:
I rather have default=error so that new users easily understand they should specify the device :)
And also maybe so that later we can have default=detected last build?